<launch>

  <rosparam command="load" file="$(find fwdis)/config/fwdis.yaml"/>
  <node pkg="fwdis" name="fwdis_controller" type="fwdis_controller"/>
  <node pkg="fwdis" name="fwdis_odom" type="fwdis_odom"/>

</launch>
